package.json

```diff
@@ -28,7 +28,7 @@
         "theme": "light"
     },
     "engines": {
-        "vscode": "1.62.0-insider"
```

Comment:
I thought this should be 1.63.0-insider

Reply:
Perhaps not, i guess its ok. Can't remember what i did.
It shouldn't matter as releases are off